sql-files/logs.sql

```diff
@@ -225,3 +225,19 @@ CREATE TABLE IF NOT EXISTS `zenylog` (
   PRIMARY KEY  (`id`),
   INDEX (`type`)
 ) ENGINE=MyISAM AUTO_INCREMENT=1;
+
+-- 
+-- Table structure for table `interlog`
+--
+
+CREATE TABLE IF NOT EXISTS `interlog` (
+	`time` DATETIME NULL DEFAULT NULL,
+	`activity` VARCHAR(255) NULL DEFAULT NULL,
```

Comment:
How about existing logs? Should we preserve them for the upgrades part of the sql?